src/apps/slackapp/slackapp/bolt_app.py

```diff
@@ -202,7 +198,7 @@ def event_test(client, say, event):
         if "files" in input_message
         else input_message["team"]
     )
-    combined_id = user_id + "_" + team_id
+    combined_id = user_id + team_id
```

Comment:
the reason behind the underscore was we could separate it and we can get the user_id to put into the whitelister. since we want to avoid platform-dependent code we should handle the whitelist in another way. which we have to discus about.

Reply:
I see. I think the underscore is OK if the rest of Sherpa treats the user ID as an opaque string.

Please read my comments about this in Linear SHX-38 or #311  and let me know there what you think.